src/lists/list-item.jsx

```diff
@@ -296,16 +296,18 @@ const ListItem = React.createClass({
   },
 
   _createTextElement(styles, data, key) {
-    const isAnElement = React.isValidElement(data);
-    const mergedStyles = isAnElement ?
-      Object.assign({}, styles, data.props.style) : null;
-
-    return isAnElement ? (
-      React.cloneElement(data, {
+    if (React.isValidElement(data)) {
+      let style = Object.assign({}, styles, data.props.style);
+      if (typeof data.type === 'string') { // if element is a native dom node
```

Comment:
> What about a custom component that is not a Material-UI one :grin:?

I'm not sure if I understand. This is really just fixing the duplicate `prepareStyles` call, and doesn't change the original logic (style prop was still passed to non Material-UI components in previous implementation). I don't think we should call `prepareStyles` on the style object for any components (Material-UI or non-Material-UI).